editor/layout/style.scss

```diff
@@ -0,0 +1,7 @@
+.editor-layout.sidebar-opened .editor-layout__content {
+	margin-right: $sidebar-width;
+
+	.editor-text-editor__formatting {
```

Comment:
I just learned that we can do `.editor-layout.is-sidebar-opened &` And yes, this is probably better!